### PR TITLE
improve truncatContentState func

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: e2e/tests/fixtures/truncateContentState.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
```
Comment (shaulgo) [inline e2e/tests/fixtures/truncateContentState.json:1]: let's move this to the test folder. it is only for the contentState spec

File: packages/common/web/src/test/contentStateServices.spec.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable max-len */
-import plain from '../../../../../e2e/tests/fixtures/plain.json';
+import raw from '../../../../../e2e/tests/fixtures/truncateContentState.json';
 import { truncateContentState } from '../lib/contentStateServices';
-const plainTop2Blocks = {
+const expectedRaw = {
```
Comment (shaulgo) [inline packages/common/web/src/test/contentStateServices.spec.js:4]: let's move this to the same file with the state. 
So we have contentStateServices.specData.js
This file can have the data, truncated0, truncated1..., truncated100

File: e2e/tests/fixtures/truncateContentState.json (hunk 1 above)
Comment (maimonav) [reply e2e/tests/fixtures/truncateContentState.json]: fixed.

File: packages/common/web/src/test/contentStateServices.spec.js (hunk 2 above)
Comment (maimonav) [reply packages/common/web/src/test/contentStateServices.spec.js]: I did it in a different way before I saw your comment, have a look and tell me if the way i did it suits you.

**Pull request merged**